WheelDeal/browse.php

```diff
@@ -3,7 +3,7 @@
 
 <div class="container">
 
-<h2 class="my-3">Browse listings</h2>

```

Comment:
shouldnt it be plural bc it's multiple listings?

Reply:
Yeah, sorry was just testing out which part of the website it was and if I was correctly connected lol so was just playing around with a h2 tag that I knew wouldn't have been affected.